<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>glaxnimate::gui::ExportImageSequenceDialog</class>
 <widget class="QDialog" name="glaxnimate::gui::ExportImageSequenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export as Image Sequence</string>
  </property>
  <property name="windowIcon">
   <iconset theme="folder-images&#9;"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>To Frame</string>
     </property>
     <property name="buddy">
      <cstring>input_frame_to</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="input_format"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>File Name</string>
     </property>
     <property name="buddy">
      <cstring>input_name</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QSpinBox" name="input_frame_to"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="input_name">
     <property name="toolTip">
      <string>Use {frame} to represent the frame number</string>
     </property>
     <property name="text">
      <string>Frame {frame}</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QSpinBox" name="input_frame_step">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Frame Step</string>
     </property>
     <property name="buddy">
      <cstring>input_frame_step</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="input_path"/>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QSpinBox" name="input_frame_from"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>From Frame</string>
     </property>
     <property name="buddy">
      <cstring>input_frame_from</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Path</string>
     </property>
     <property name="buddy">
      <cstring>input_path</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="button_select">
     <property name="text">
      <string>Select...</string>
     </property>
     <property name="icon">
      <iconset theme="folder-open">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QProgressBar" name="progress_bar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>input_path</tabstop>
  <tabstop>button_select</tabstop>
  <tabstop>input_name</tabstop>
  <tabstop>input_format</tabstop>
  <tabstop>input_frame_from</tabstop>
  <tabstop>input_frame_to</tabstop>
  <tabstop>input_frame_step</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>glaxnimate::gui::ExportImageSequenceDialog</receiver>
   <slot>render()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_select</sender>
   <signal>clicked()</signal>
   <receiver>glaxnimate::gui::ExportImageSequenceDialog</receiver>
   <slot>pick_path()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>glaxnimate::gui::ExportImageSequenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>render()</slot>
  <slot>pick_path()</slot>
 </slots>
</ui>
